common/djangoapps/student/views.py

```diff
@@ -2552,13 +2556,13 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
 
     if request.method == 'POST':
         password = request.POST['new_password1']
-        is_password_valid, password_err_msg = validate_password(user, password)
-        if not is_password_valid:
+        is_security_violated, password_err_msg = validate_password(user, password)
+        if password_err_msg:
             # We have a password reset attempt which violates some security
-            # policy. Use the existing Django template to communicate that
+            # policy, or minimum length. Use the existing Django template to communicate that
             # back to the user.
             context = {
-                'validlink': False,
+                'validlink': not is_security_violated,
```

Comment:
Why can't we say `validlink` is also `False` when passwords do not meet the `length` validation?